acp/main_module.php

```diff
@@ -67,11 +70,12 @@ public function __construct()
 		global $phpbb_container;
 
 		$this->container   = $phpbb_container;
-		$this->cache       = $this->container->get('cache');
+		$this->cache       = $this->container->get('cache.driver');
 		$this->config      = $this->container->get('config');
 		$this->config_text = $this->container->get('config_text');
 		$this->language    = $this->container->get('language');
 		$this->log         = $this->container->get('log');
+		$this->mediaembed_cache = $this->container->get('phpbb.mediaembed.purge_cache');
```

Comment:
Sorry to be going in circles on this but now that I see it, lets call this service `phpbb.mediaembed.cache`

Let's just consider this class an object for handling anything cache related, though for now it's just purging. So the class would be named `phpbb\mediaembed\cache\cache` and it's only method right now will be the `purge` one.